libs/service-portal/finance/src/screens/FinanceLoans/FinanceLoans.graphql

```diff
@@ -0,0 +1,81 @@
+query GetHmsLoansLoanhistory {
+  hmsLoansLoanhistory {
```

Comment:
lowercase h owie 🤕

Reply:
Also, you don't have to repeat the same word in singular/plural while naming the queries. `HmsLoansHistory` should cover it. If it doesn't, might we worth a rethink on the modelling, i think